fvm/blueprints/contractDeployment.go

```diff
@@ -0,0 +1,38 @@
+package blueprints
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const ContractDeploymentAuthorizedAddressesPathDomain = "storage"
+const ContractDeploymentAuthorizedAddressesPathIdentifier = "authorizedAddressesToDeployContracts"
+
+const setContractDeploymentAuthorizersTransactionTemplate = `
+transaction {
+		prepare(signer: AuthAccount) {
+			// this drops previous list
+			signer.load<[Address]>(from: /%s/%s)
+			signer.save([%s], to: /%s/%s)
+		}
+}
```

Comment:
Instead of string formatting the details, pass them as arguments:

```suggestion
transaction(addresses: [Address], path: StoragePath) {
		prepare(signer: AuthAccount) {
			signer.load<[Address]>(from: path)
			signer.save(addresses, to: path)
		}
}
```

Reply:
You can prepare the arguments just like you handle the result of the read in `OptionalCadenceValueToAddressSlice` (just that it shouldn't be optional)

Reply:
fixed at 9cb4197